lizmap/www/assets/css/main.css

```diff
@@ -35,7 +20,8 @@ a:hover {
   z-index: 1030;
   margin-bottom: 0px;
   overflow: visible;
-  background : url('img/header-background-medium.jpg') repeat fixed #2B2B2B;
+  background-color: #2B2B2B;
+  background-image : url(img/header-background-medium.jpg);
```

Comment:
I would add a background-repeat and the other config to set the "fixed" property, to be sure on big screens, there is no blank (the image must be repeated)

Reply:
If I set `background-attachment : fixed`, the black color goes above image. Default is `background-attachment : scroll`. As there is no scroll in the element I think we should let default.

Reply:
I've added `background-repeat: repeat;`.